src/blocks/owc-openkaarten/streetmap/assets/scripts/vue/App.vue

```diff
@@ -234,7 +234,9 @@ onMounted(() => {
 
 .owc-openkaarten-streetmap-container {
   margin-inline: auto;
-  width: min(calc(100% - 32px), 1440px);
```

Comment:
Waarom is hier niet gekozen voor max-width? Dan hoeft dit niet in een media-query.